src/backends/interflop-mca/interflop_mca.c

```diff
@@ -106,6 +113,16 @@ typedef enum {
 
 static const char *MCA_MODE_STR[] = {"ieee", "mca", "pb", "rr"};
 
+/* define the available error modes */
+typedef enum {
+  mca_err_mode_rel,
+  mca_err_mode_abs,
+  mca_err_mode_all,
+  _mca_err_mode_end_
```

Comment:
Why is this canary (`_mca_err_mode_end`) needed ?

Reply:
It's not actually needed.
But it made sense to have it, in order to have the same coding style as in the rest of the file.

Reply:
I've removed it.
From interflop_vprec, as well.